lib/internal/abort_controller.js

```diff
@@ -435,7 +436,8 @@ async function aborted(signal, resource) {
   if (signal.aborted)
     return PromiseResolve();
   const abortPromise = createDeferredPromise();
-  signal.addEventListener('abort', abortPromise.resolve, { [kWeakHandler]: resource, once: true });
+  const opts = { __proto__: null, [kWeakHandler]: resource, once: true, [kResistStopPropagation]: true };
```

Comment:
is `kResistStopPropagation` still needed?

Reply:
This function uses `kWeakHandler` which is not available in `addAbortListener`, so here we still need it, yes...